extensions/smallrye-health/deployment/pom.xml

```diff
@@ -34,6 +34,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-health-spi</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Does it work if the OpenAPI SPI dependency is optional?

Reply:
It does !